Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -61,6 +61,8 @@ namespace RealmWeaver
     // - An anonymized MAC address and assembly name ID to aggregate the other information on.
     internal class Analytics
     {
+        private const string _coreVersion = "";
```

Comment:
We can probably fill in the current Core version since the expectation is that there'll be something here.

Reply:
But then we don't fully appreciate the regex that works with an empty string too 😢